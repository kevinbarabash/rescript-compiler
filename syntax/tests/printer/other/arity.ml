let _ = Constructor (a, b)
let _ = `Constructor (a, b)

let _ = match a with
| C (c, d) -> 1

let _ = match a with
| `C (c, d) -> 1
