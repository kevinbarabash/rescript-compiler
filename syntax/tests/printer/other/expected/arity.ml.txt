let _ = Constructor(a, b)
let _ = #Constructor(a, b)

let _ = switch a {
| C(c, d) => 1
}

let _ = switch a {
| #C(c, d) => 1
}
